Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -49,10 +49,10 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.goobi.production.cli.helper.WikiFieldHelper;
+import org.goobi.production.plugin.PluginLoader;
```

Comment:
Maybe you should not move this import statement to an non-ordered list and then you must not increase allowed violations.